docs/user-guide.rst

```diff
@@ -329,6 +329,155 @@ Follow the detailed instructions on how to
 .. _Configure Workload Identity Federation from an OIDC identity provider:
     https://cloud.google.com/iam/docs/access-resources-oidc
 
+Using Executable-sourced credentials with OIDC and SAML
```

Comment:
This is formatted incorrectly (has extra /* everywhere)

Reply:
Do you mean "/*" is missing everywhere? There is no "/*" in the current version.